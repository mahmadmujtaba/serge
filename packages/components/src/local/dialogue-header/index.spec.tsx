/* global it expect */

import React from 'react'
import renderer from 'react-test-renderer'

import DialogueHeader from './index'

it('DialogueHeader renders correctly', () => {
  const tree = renderer
    .create(<DialogueHeader string="test" />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
