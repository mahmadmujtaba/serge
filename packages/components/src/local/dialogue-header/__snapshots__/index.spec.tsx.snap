// Jest Snapshot v1, https://goo.gl/fbAQLP

exports['DialogueHeader renders correctly 1'] = `
<h2
  className="dialogue-header"
>
  test
</h2>
`
